src/component/plotly-graph/parse.js

```diff
@@ -155,11 +167,23 @@ function willFigureHang (result) {
 // can be (much) smaller than the true number of points plotted
 // when it does not match the length of the other coordinate arrays.
 function findMaxArrayLength (cont) {
-  const lengths = Object.keys(cont)
+  const arrays = Object.keys(cont)
     .filter(k => Array.isArray(cont[k]))
-    .map(k => cont[k].length)
+    .map(k => cont[k])
 
-  return Math.max(...lengths)
+  const lengths = arrays.map(arr => {
+    const innerArrays = arr.filter(Array.isArray)
+
+    if (innerArrays.length) {
+      return innerArrays
+        .map(a => a.length)
+        .reduce((a, v) => a + v)
+    } else {
+      return arr.length
+    }
+  })
+
+  return arrays.length ? Math.max(...lengths) : 0
```

Comment:
🐄 not that it matters, but `Math.max(0, ...lengths)` works too

Reply:
:cowboy_hat_face: -> 851b803